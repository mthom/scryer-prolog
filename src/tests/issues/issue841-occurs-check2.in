set_prolog_flag(occurs_check, true).
X = -X.
asserta(f(X,g(X))).
f(X,X).
X-X = X-g(X).
halt.
