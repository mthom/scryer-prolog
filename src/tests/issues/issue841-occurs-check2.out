   true.
   false.
   true.
   false.
   false.
